panel/src/components/Forms/Counter.vue

```diff
@@ -1,9 +1,12 @@
 <template>
 	<span :data-invalid="!valid" class="k-counter">
 		<span>{{ count }}</span>
-		<span v-if="min && max" class="k-counter-rules">({{ min }}–{{ max }})</span>
-		<span v-else-if="min" class="k-counter-rules">≥ {{ min }}</span>
-		<span v-else-if="max" class="k-counter-rules">≤ {{ max }}</span>
+
+		<span class="k-counter-rules">
```

Comment:
Shouldn't the span still be removed if min and max are not set?

Reply:
True